runtime/vm/classsupport.c

```diff
@@ -568,8 +568,7 @@ callFindLocallyDefinedClass(J9VMThread* vmThread, J9Module *j9module, U_8* class
 			if (returnVal != NULL) {
```

Comment:
Isn't there a problem above where TRIGGER_J9HOOK_VM_FIND_LOCALLY_DEFINED_CLASS passes the classPathEntries and classLoader->classPathEntryCount ?

Reply:
I suggest NULL, 0 should be passed instead.

Reply:
These two parameters are ignored by `hookFindSharedClass()` if it is bootstrap class loader. We will always read them from bootstrap classloader inside the mutex there.

Reply:
Yes, but GAC's saying the same thing elsewhere, the code should pass NULL, 0, to ensure they are not used.

Reply:
Right, I'd rather make the decision in the caller rather than the listener(s) as it will avoid any future problems if new listeners are added. It also allow the local override of the classLoader data for all loaders if that were ever to be required.

Reply:
> but GAC's saying the same thing elsewhere, the code should pass NULL, 0,

Yes, this can be done. I will change to pass in NULL and 0.

Reply:
Please also update the doc in the hook declaration to describe the new contract.